qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/services/QuteCodeActions.java

```diff
@@ -133,6 +141,46 @@ public CompletableFuture<List<CodeAction>> doCodeActions(Template template, Code
 		return CompletableFuture.completedFuture(codeActions);
 	}
 
+	private void doCodeActionToCreateProperty(Template template, Diagnostic diagnostic, QuteErrorCode errorCode,
+			List<CodeAction> codeActions, QuteTemplateGenerateMissingJavaMember resolver, SharedSettings settings) {
+		JsonObject data = (JsonObject) diagnostic.getData();
+		String missingProperty = data.get("property").getAsString();
+		String resolvedType = data.get("signature").getAsString();

```

Comment:
Dont use string for property like "signature" but declare a static method where you create the JsonObject. See a sample inlemminx